doc/sphinx/practical-tools/coq-commands.rst

```diff
@@ -226,6 +233,89 @@ and ``coqtop``, unless stated otherwise:
   currently associated color and exit.
 :-h, --help: Print a short usage and exit.
 
+
+
+Compiled interfaces (produced using `-vos`)
+----------------------------------------------
+
+
+Compiled interfaces helps saving time while developing Coq formalizations,
+by compiling the formal statements exported by a library independently of
+the proofs that it contains.
+
+   .. warning::
+
+      Compiled interfaces should only be used for development purposes.
+      At the end of the day, one still needs to proof check all files
+      by producing standard `.vo` files. (Technically, when using `-vos`,
+      fewer universe constraints are collected.)
+
+   .. warning::
+
+      In the current implementation, the use of the command `Include` for
+      importing modules compiled using `-vos` might not work properly.
+
+
+**Typical usage.**
+
+Assume a file ``foo.v`` that depends on two files ``bar1.v`` and ``bar2.v``. The
+command ``make foo.requires_vos`` will compile ``bar1.v`` and ``bar2.v`` using
+the option ``-vos`` to skip the proofs, producing ``bar1.vos`` and ``bar2.vos``.
+At this point, one is ready to work interactively on the file ``foo.v``, even
+though it was never needed to compile the proofs involved in the ``bar*.v``
+files.
+
+Assume a set of files ``f1.v ... fn.v`` with linear dependencies. The command
+``make vos`` enables compiling the statements (i.e. excluding the proofs) in all
+the files. Next, ``make -j vok`` enables compiling all the proofs in parallel.
```

Comment:
Again, that's something supported by `coq_makefile`?

Reply:
I have update coq_makefile generator to support this, indeed.

Reply:
Maybe it should be made explicit in the doc.